tuplex/python/src/PythonContext.cc

```diff
@@ -39,35 +39,35 @@ namespace tuplex {
 
         Schema schema(Schema::MemoryLayout::ROW, python::Type::makeTupleType({python::Type::F64}));
 
-        std::vector<std::tuple<size_t, PyObject*>> badParallelizeObjects;
-        std::vector<size_t> numExceptionsInPartition;
-        
         // check if empty?
         if(0 == numElements)
-            return _context->fromPartitions(schema, std::vector<Partition*>(), columns, badParallelizeObjects, numExceptionsInPartition);
+            return _context->fromPartitions(schema, std::vector<Partition*>(), std::vector<Partition*>(), std::vector<PartitionGroup>(), columns);
 
         // create new partition on driver
         auto driver = _context->getDriver();
 
+        std::vector<std::tuple<size_t, PyObject*>> fallbackRows;
```

Comment:
some doc maybe what this variables contain? specifically for the fallback rows I assume it's some index or so?